### ISSUE TITLE
Unable to view or download script package

### ISSUE THREAD

**Issue by Runax15** [rgbkrk/atom-script#2678]
![image](https://user-images.githubusercontent.com/85685609/166933161-794ca53a-3df9-4cf9-b14f-ff57e53da4c1.png)
I'm unable to view script package after searching for it. Also not able to download it from online website.

**Comment by Gledders1G1** [comment]
I have the same problem, have you managed to find a fix/work around?

**Comment by gitUmaru** [comment]
I had the same issue where it was not showing up on the package page inside atom. Their travis CI build status also seems to be down right now, so perhaps there's an issue internally right now? Not sure.

I installed via `apm` and that worked for me on a new machine.

`apm install script`

**Comment by shinysocks** [comment]
Same issue here.
in-app package manager does not show option for script:
![Screenshot from 2022-05-07 09-08-52](https://user-images.githubusercontent.com/91911303/167257875-0e5b7cd7-5a98-4998-b0bf-0814f644b790.png)
![pix](https://user-images.githubusercontent.com/91911303/167257824-c78c900f-accd-4e56-8a4c-52c6b066c99c.png)

installed successfully using 
`apm install script`

**Comment by Runax15** [comment]
> I installed via `apm` and that worked for me on a new machine.
> 
> `apm install script`

Thank You i was able to download the script package with your help.

**Comment by Runax15** [comment]
Thanks the issue is resolved with your help.

**Comment by Gledders1G1** [comment]
sorry I'm new to coding, could someone give a detailed map of how to get install script this way?

**Comment by KokoCC20** [comment]
Any solutions for window? Package isn't showing and I really want to try Atom on here.

**Comment by gitUmaru** [comment]
Yes, you can install script via the apm package manager in Windows. If you are unfamiliar with this process, then just follow these steps:

1. Open a terminal so you can use the command line
    * This can be Power Shell, Command Prompt, or even Git Bash!
2. Type in `apm -v` to print the current version (this is to ensure that apm is installed)
    * If apm is not installed then you can try reinstalling Atom
    * If this still doesn't work, then apm might not be added to your Path. Follow this [thread here](https://github.com/atom/apm/issues/278)
3. Type `apm install script`
4. Restart Atom

**Comment by Gledders1G1** [comment]
thanks for the fix! helped my beginner ass alot!

**Comment by KokoCC20** [comment]
Hmm, I feel dumb, I assumed apm was a linux thing only but found out it is installed with atom. Thanks, you response was the fix needed and it helped a lot.

**Comment by Snakeynaut** [comment]
> Yes, you can install script via the apm package manager in Windows. If you are unfamiliar with this process, then just follow these steps:
> 
>     1. Open a terminal so you can use the command line
>        
>        * This can be Power Shell, Command Prompt, or even Git Bash!
> 
>     2. Type in `apm -v` to print the current version (this is to ensure that apm is installed)
>        
>        * If apm is not installed then you can try reinstalling Atom
>        * If this still doesn't work, then apm might not be added to your Path. Follow this [thread here](https://github.com/atom/apm/issues/278)
> 
>     3. Type `apm install script`
> 
>     4. Restart Atom

Thanks a bunch Umar! I've made a video tutorial on fixing this issue [here](https://www.youtube.com/watch?v=dQw4w9WgXcQ)
Lemme know if it is accurate.

**Comment by fauzymadani** [comment]
i tried to run "apm install script" but still can't download that,failed.it showing this:
Request for package information failed: Hostname/IP does not match certificate's altnames: Host: atom.io. is not in the cert's altnames: DNS:*.tri.co.id, DNS:tri.co.id (ERR_TLS_CERT_ALTNAME_INVALID)

**Comment by senthur123** [comment]
it won't work as the repository https://atom.io/packages/script itself does not exists since the website https://atom.io not exists. it has got sunsetted and getting redirected to github. The URL handler points to atom:// but could not find the url path. If anyone knows the alternate path to https://atom.io/packages/script pls let us all know.
